fcrepo-webapp/src/main/webapp/static/constraints/owl2html.xsl

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Additionally, this stylesheet tries to do much more than is needed by the constraint RDF documents. See attached screenshot for sections that should be removed from this xsl.
![screenshot from 2015-06-22 16 32 32](https://cloud.githubusercontent.com/assets/149283/8292206/50cf36a8-18fc-11e5-9d9a-e5e74cfbeace.png)